amundsen_application/static/js/components/common/Badges/styles.scss

```diff
@@ -1,10 +1,25 @@
 // Copyright Contributors to the Amundsen project.
 // SPDX-License-Identifier: Apache-2.0
-
 @import 'variables';
-
 .clickable-badge {
   &:hover {
     cursor: pointer;
   }
-}
+  .label {
+    overflow: hidden;
+    padding: 0 0;
+  }
+  .badge-overlay-primary {
+    font-size: 14px;
+    text-align: center;
+    display: inline-block;
+    padding: .2em .6em .3em;
+  }
+  .badge-overlay-primary:focus, .badge-overlay-primary:hover {
+    background-color: rgba($color: $badge-overlay, $alpha: .14);
+    color: $text-primary;
+    height: 20px;
```

Comment:
I would store this into a local variable, something like $badge-height or similar